src/views/AddItem.jsx

```diff
@@ -5,25 +5,34 @@ export function AddItem() {
 	const [newItem, setNewItem] = useState('');
 	const [nextPurchaseTime, setPurchaseTime] = useState(7);
 	const [statusMessage, setStatusMessage] = useState('');
+	const [alertVisible, setAlertVisible] = useState(false);
 
 	const handleSubmit = async (e) => {
 		e.preventDefault();
 
 		let listId = 'my test list';
 
+		console.log(nextPurchaseTime, newItem);
 		const item = await addItem(listId, {
 			itemName: newItem,
 			daysUntilNextPurchase: nextPurchaseTime,
 		});
 
 		if (item.id) {
-			setStatusMessage('Item Successfully Added!');
+			setAlertVisible(true);
+			setStatusMessage(`${newItem} Successfully Added!`);
+
 			setNewItem('');
 			setPurchaseTime(7);
+
+			setTimeout(() => {
+				setAlertVisible(false);
+			}, 3000);
```

Comment:
you can clear it directly here without having an additional state. just replace `setAlertVisible(false)` with `setStatusMessage("")`